cmd/minishift/cmd/config/view.go

```diff
@@ -47,11 +50,17 @@ var configViewCmd = &cobra.Command{
 	Short: "Display the properties and values of the Minishift configuration file.",
 	Long:  "Display the properties and values of the Minishift configuration file. You can set the output format from one of the available Go templates.",
 	Run: func(cmd *cobra.Command, args []string) {
-		err := configView()
+		cfg, err := ReadConfig()
```

Comment:
We really need to remove the ''\n" in the default format. It affects the output of the help text, breaking the line. I'll post some code in a second.